backend/src/api/api.ts

```diff
@@ -142,6 +144,15 @@ const initApi: FastifyPluginAsync = async (fastify) => {
     },
     prefix: ApiPath.TASKS,
   });
+
+  fastify.register(initBillingApi, {
+    services: {
+      billing,
+      user,
+      userDetails,
+    },
+    prefix: ApiPath.BILLING,
+  });
```

Comment:
```suggestion
  fastify.register(initBillingApi, {
    services: {
      billing,
    },
    prefix: ApiPath.BILLING,
  });
```
You don't seem to use them in api